src/libraries/System.Private.CoreLib/src/System/DateTime.Windows.cs

```diff
@@ -111,12 +155,14 @@ internal FullSystemTime(long ticks)
             }
         }
 
-#if !CORECLR
-        internal static readonly bool s_systemSupportsPreciseSystemTime = SystemSupportsPreciseSystemTime();
+        private static unsafe readonly delegate* unmanaged[SuppressGCTransition]<long*, void> s_pfnGetSystemTimeAsFileTime = GetGetSystemTimeAsFileTimeFnPtr();
```

Comment:
I admit when support was added for function pointers I was not expecting us to be able to make such extensive use of it. Cool.